client/client-multi/client-cpp/impl/context/AbstractExecutorContext.cpp

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#include "kaa/logging/Log.hpp"
```

Comment:
Sorry, for the mess, but my code doesn't compile with this header, so I leave it.